src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/SummaryEvidence.cshtml

```diff
@@ -46,16 +46,16 @@
                     <caption class="govuk-table__caption govuk-visually-hidden">Obligation evidence summary</caption>
                     <thead class="govuk-table__head">
                         <tr class="govuk-table__row">
-                            <th class="govuk-table__header">
+                            <th class="govuk-table__header govuk-!-text-align-left" style="width: 1%;">
```

Comment:
@meglenapnovoselska is 1% width here correct, think that probably needs to reflect the actual % width of the column

Reply:
@JohnPadgettCivica agree, it's changed.